cli/githubutil.py

```diff
@@ -146,18 +150,110 @@ def list_draft_releases(
     github_cfg_name: str,
     github_repository_owner: str,
     github_repository_name: str,
+    greatest_release_version: CliHint(
+        help=(
+            'Greatest version (according to Semver) up to which (exclusive) '
+            'to consider draft releases, if outdated draft releases are to be determined'
+        ),
+    )=None,
+    list_outdated_draft_releases: bool = False,
 ):
+    '''List all draft releases in a GitHub repository. Optionally, attempt to print only outdated
+    draft releases.
+
+    This function prints a list of names of draft releases found in the given GitHub repository.
+    If the `--list-outdated-draft-releases` flag is set, the printed list is reduced to outdated
```

Comment:
please do not add irrelevant implementation details to the documentation. it is irrelevant that the releases are added into a list.

state that the flag will heuristically filter all "still relevant" drafts releases, thus outputting draft releases that are considered to be obsolete (and thus subject to being cleaned up).